tests/logger/test_metrics_context.py

```diff
@@ -258,6 +287,28 @@ def test_put_metric_uses_none_unit_if_not_provided():
     assert metric.unit == "None"
 
 
+@pytest.mark.parametrize(
+    "name, value, unit",
+    [
+        ("", 1, "None"),
```

Comment:
If the user passes `put_metric("Example", 1.0, None)`

(No quotes around `None`).

What happens? That should pass, right?

Reply:
Yes, that will pass. Using `None` as a metric unit will set it to `"None"` in the metric constructor `metric.py`.

Reply:
Awesome